pkgs/applications/networking/avalanchego/default.nix

```
@@ -0,0 +1,41 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, stdenv
+, IOKit
+}:
+
+buildGoModule rec {
+  pname = "avalanchego";
+  version = "1.9.5";
+
+  src = fetchFromGitHub {
+    owner = "ava-labs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-Uim2Q3LvNEV0C5/gFdAiXKy2oKbhhc/ifRyz5BIh6zE=";
+  };
+
+  # go mod vendor has a bug, see: https://github.com/golang/go/issues/57529
+  proxyVendor = true;
+
+  subPackages = [ "main" ];
+
+  vendorHash = "sha256-43bUvEwuppizLybj32eolSdpnqGJkkdA53/uBZkCrDc=";
```

```suggestion
  vendorHash = if stdenv.isDarwin
    then "sha256-B3QC5IVVqyN59phG51dVm4glGZzLYt1mmlAxZXNsqTE="
    else "sha256-43bUvEwuppizLybj32eolSdpnqGJkkdA53/uBZkCrDc=";
```

We now have two vendorHashes because of two different zondax/hid dependencies.